ovs/extensions/storageserver/storagedriver.py

```diff
@@ -433,3 +433,9 @@ def _delete(self, section):
         """
         if section in self.configuration:
             del self.configuration[section]
+
+def is_clusterNotReachableException(exception):
+    if isinstance(exception, ClusterNotReachableException) or (isinstance(exception, RuntimeError) and 'failed to send XMLRPC request' in str(exception)):
+        return True
```

Comment:
`return isinstance(exception, ClusterNotReachableException) or (isinstance(exception, RuntimeError) and 'failed to send XMLRPC request' in str(exception)):` already provide the True - False. No need to check it again